packages/core/test/ethpm.js

```diff
@@ -1,314 +1,409 @@
 var assert = require("chai").assert;
```

Comment:
btw if you're looking to test this locally, I usually just create an alias in my shell:
```bash
alias truffle="path/to/truffle/packages/core/cli.js"
```

Or, to test the bundled version (the version that everyone uses)
```bash
yarn bootstrap
alias truffle="node /path/to/truffle/packages/truffle/build/cli.bundled.js"
```